src/main/kotlin/no/nav/syfo/kafka/consumers/spleis/domain/UtbetalingSpleis.kt

```diff
@@ -17,7 +18,54 @@ data class UtbetalingSpleis(
     val tom: String,
     val utbetalingId: String,
     val korrelasjonsId: String,
+    val utbetalingdager: List<UtbetalingdagDto> = emptyList(),
 )
 
+@JsonIgnoreProperties(ignoreUnknown = true)
+data class UtbetalingdagDto(
+    val dato: LocalDate,
+    val type: DagType,
+    val begrunnelser: List<String>,
+)
+
+enum class DagType {
+    ArbeidsgiverperiodeDag,
+    NavDag,
+    NavHelgDag,
+    Arbeidsdag,
+    Fridag,
+    AvvistDag,
+    UkjentDag,
+    ForeldetDag,
+    Permisjonsdag,
+    Feriedag,
+    ArbeidIkkeGjenopptattDag,
+    AndreYtelser,
+    Venteperiodedag,
+}
+
+/**
+Begrunnelse:
+AndreYtelserAap,
+AndreYtelserDagpenger,
+AndreYtelserForeldrepenger,
+AndreYtelserOmsorgspenger,
+AndreYtelserOpplaringspenger,
+AndreYtelserPleiepenger,
+AndreYtelserSvangerskapspenger,
+SykepengedagerOppbrukt,
+SykepengedagerOppbruktOver67,
+MinimumInntekt,
+EgenmeldingUtenforArbeidsgiverperiode,
+MinimumSykdomsgrad,
+ManglerOpptjening,
+ManglerMedlemskap,
+EtterDødsdato,
+Over70,
+MinimumInntektOver67,
+NyVilkårsprøvingNødvendig,
+UKJENT
+ **/
```

Comment:
Skal dette gjøres om til en enum eller noe lignende? 😊

Reply:
Kunne det vært ryddigere å brukt KDoc på typen som holder på propertien? 😊 Det er kanskje `UtbetalingdagDato`? Så kunne vi i stedet for å ramset opp alle her linket til der de er definert som kanskje er i `helse-sporbar`?